snippets/csharp/System.Reflection.Metadata/SequencePoint/Overview/Program.cs

```diff
@@ -8,8 +8,15 @@ class Program
     {
         static void Main(string[] args)
         {
-            string pdbPath = Path.Combine(Path.GetDirectoryName(typeof(Program).Assembly.Location), "SequencePointSnippets.pdb");
-            SequencePointSnippets.ReadSourceLineData(pdbPath, MethodInfo.GetCurrentMethod().MetadataToken);
+            string? loc = Path.GetDirectoryName(typeof(Program).Assembly.Location);
+            if (loc is not null)
```

Comment:
Same comment as earlier:

```suggestion
            if (Path.GetDirectoryName(typeof(Program).Assembly.Location is string loc)
```